index.html

```diff
@@ -73,6 +73,238 @@ <h4>Glad to see you!</h4>
           </div>
         </div>
       </section>
+
+      <!-- Work Section -->
+      <section class="project-container">
+        <div class="project">
+          <div class="grid-item">
+            <img
+              class="portfolio-snapshot"
+              src="./images/snapshot.png"
+              alt="portfolio-snapshot"
+            />
+          </div>
+          <div class="left-block">
+            <div class="project-details">
+              <h2 class="project-title">Tonic</h2>
+              <ul class="project-stack">
+                <li><h3 class="project-sub-title">CANOPY</h3></li>
+                <li><img src="./images/counter.png" alt="counter" /></li>
+                <li><h4 class="project-year-role">Back End Dev</h4></li>
+                <li><img src="./images/counter.png" alt="counter" /></li>
+                <li><h4 class="project-year-role">2015</h4></li>
+              </ul>
+            </div>
+            <div>
+              <p class="project-desc">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div>
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
+            </div>
+            <div class="project-action-btn">
+              <button class="detail-btn">See project</button>
+            </div>
+          </div>
+        </div>
+        <div class="project">
+          <div class="grid-item">
+            <img
+              class="portfolio-snapshot"
+              src="./images/snapshot-2.png"
+              alt="portfolio-snapshot"
+            />
+          </div>
+          <div class="left-block">
+            <div class="project-details">
+              <h2 class="project-title">Multi-Post Stories</h2>
+              <ul class="project-stack">
+                <li><h3 class="project-sub-title">CANOPY</h3></li>
+                <li><img src="./images/counter.png" alt="counter" /></li>
+                <li><h4 class="project-year-role">Back End Dev</h4></li>
+                <li><img src="./images/counter.png" alt="counter" /></li>
+                <li><h4 class="project-year-role">2015</h4></li>
+              </ul>
+            </div>
+            <div>
+              <p class="project-desc">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div>
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
+            </div>
+            <div class="project-action-btn">
+              <button class="detail-btn">See project</button>
+            </div>
```

Comment:
- [ ] Kindly ensure that you follow suit with all the design guidelines to achieve an optimized pixel-perfect design like you'll be doing in your next job. Note that everything must much as possible, the font size, font weight, margin, padding, background image, color, line height, font family, and the like.

- Your design
<img width="155" alt="wrong" src="https://user-images.githubusercontent.com/41163571/228017626-216492be-664f-4d71-acb4-706c29d30ea4.png">

- Figma design
<img width="198" alt="right" src="https://user-images.githubusercontent.com/41163571/228017684-d3425b35-1737-45c5-a9be-1d7b9f98340a.png">

- The first one should have the same color as the design and the font size should be a little bit reduced.
- For the button the text in it likewise the border-size should be bolder. 

- You can always get the properties on Figma by clicking on the item, and then check the right side pane for the information.